src/kits/tracker/ContainerWindow.cpp

```diff
@@ -1010,13 +968,41 @@ BContainerWindow::Init(const BMessage* message)
 	SetFlags(Flags() & ~B_NO_WORKSPACE_ACTIVATION);
 }
 
+void
+BContainerWindow::InitLayouts()
```

Comment:
IMO this should be called InitLayout(), or LayoutViews(). A plural of "layout" just sounds wrong to me, at least.